client/src/composables/uuid.ts

```diff
@@ -0,0 +1,35 @@
+import Vue, { ref } from "vue";
```

Comment:
I do not like the naming of this composable as it does not return an uuid. It also seems to have large overlap with the existing useUid composable. Would it be possible to use that instead of making a new composable?

Reply:
Yep, this should just be the existing `useUid`, it already does exactly what was discussed in wg-uiux.

Reply:
Sorry, I somehow completely missed that one, using that one instead